geonode/base/api/serializers.py

```diff
@@ -552,6 +552,22 @@ def to_representation(self, instance):
         return ret
 
 
+class ResourceSettingsField(serializers.BooleanField):
```

Comment:
Sorry for being picky, but probably this class should be named "ResourceManagementField". This will more in line with the recent discussions we had about these fields.